src/core/shape_inference/include/squeeze_shape_inference.hpp

```diff
@@ -28,6 +28,9 @@ std::vector<TRShape> shape_infer(const Squeeze* op,
     using DimType = typename T::value_type;
 
     const auto number_of_inputs = input_shapes.size();
+    if (number_of_inputs == 1) {
```

Comment:
This limitation is not according specification the number of inputs can be 1 or 2. The issue should be fixed at different stage.